package.json

```diff
@@ -19,23 +20,27 @@
   "author": "troxler",
   "files": [
     "dist",
-    "readme.md"
+    "readme.md",
+    "types"
   ],
   "main": "dist/vue-headful.js",
+  "types": "types",
   "repository": "github:troxler/vue-headful",
   "scripts": {
-    "dev": "(cd demo; poi)",
```

Comment:
It was actually intentional to start a subshell for this. Because otherwhise you won't be able to execute that command repeatedly after pressing ctrl+c as your current working directory would change.

Reply:
The thing is that such a thing doesn't work in Windows.